lib/ibge/reader/helpers.rb

```diff
@@ -0,0 +1,75 @@
+module Ibge
+  module Reader
+    module Helpers
+      SOURCES = {
+        dtb: "DTB_2014_subdistrito.xls",
+        sidra: "sidra.xls"
+      }
+
+      attr_reader :defined_columns, :selected_source
+
+      def columns(columns)
+        if columns.empty?
+          raise ArgumentError,
+            "Columns must be provided as hash which contains the index and field name"
```

Comment:
Line is too long. [86/80]<br>Align the parameters of a method call if they span more than one line.